types/decimal.go

```diff
@@ -198,7 +196,6 @@ func MustNewDecFromStr(s string) Dec {
 	return dec
 }
 
-//______________________________________________________________________________________________
```

Comment:
These line changes (and this whole commit afd8a76bd9bea82435df5a45d2e62d6b2275ec61) seems completely absurd. If anyone has an idea how to fix CI, feel free to suggest something else.

Reply:
i have even more "Unchanged files with check annotations" lint errors...